modules/bookStorage.js

```diff
@@ -0,0 +1,41 @@
+export default class BookStorage {
+  data = []
+
+  addData(book) {
+    this.data.push(book);
```

Comment:
- [ ] [OPTIONAL] Instead of using `push` which mutates the `data`, kindly favor the use array methods that don't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) 👍🏽